### DISCUSSION TITLE
Use the same class on different colections

### DISCUSSION THREAD

**Discussion by Macacoazul01** [isar/isar#733] [Help]
Is it possible to use  `isar.writeTxn(() async {})` for two (or more) different collections of the same Schema without having to create another Isar instance?

The equivalent of having two different hive boxes for the same class

**Comment by Macacoazul01** [comment]
@leisim

**Comment by simc** [comment]
I'm not sure I understand the question. Are you asking whether you can access different collections in the same transaction? You can.

**Reply by xOldeVx** [reply]
@simc for example, I have object called **Item**, and i want to add it to **favorites** and to **history** db, how can it done?
It's bad way:

```
final dir = await getApplicationDocumentsDirectory();
final favorites = await Isar.open([ItemSchema], directory: dir.path);
final history = await Isar.open([ItemSchema], directory: dir.path);
```

What do you suggest?
